lib/libutee/include/pta_invoke_tests.h

```diff
@@ -73,5 +73,23 @@
  */
 #define PTA_INVOKE_TESTS_CMD_LOCKDEP		8
 
+#define PTA_INVOKE_TESTS_MODE_ECB		0
+#define PTA_INVOKE_TESTS_MODE_CBC		1
+#define PTA_INVOKE_TESTS_MODE_CTR		2
+#define PTA_INVOKE_TESTS_MODE_XTS		3
+#define PTA_INVOKE_TESTS_MODE_GCM		4
+
+/*
+ * AES performance tests
+ *
+ * [in]     value[0].a	Top 16 bits Decrypt, low 16 bits key size in bytes
+ * [in]     value[0].b	Mode, one of TEE_MODE_{ECB_NOPAD,CBC_NOPAD,CTR,XTS,GCM}
```

Comment:
`AES mode, one of PTA_INVOKE_TESTS_AES_MODE_{ECB,CBC,CTR,XTS,GCM}`

Reply:
I'll replace the MODE part with AES, that should be clear enough.